common/changes/office-ui-fabric-react/ahackett-listheaderexpand_2019-01-14-22-33.json

```
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "office-ui-fabric-react",
+      "comment": "add aria-expanded to the DetailList column header",
```

Can you please modify the comment to start with `DetailsList: ... anything you change here...`. This comment is part of the release notes so we wanna have easy to scan notes when looking at each release change.
Thanks!
```suggestion
"comment": "DetailsList: add isMenuOpen property on IColumn interface and add aria-expanded to the DetailsList column header.",
```

---

Done. Thanks!